core/board/board.go

```diff
@@ -157,6 +174,23 @@ func (b *Board) getNeighbors(pt *point.Point) []*point.Point {
 	return points
 }
 
+// GetFullBoardState returns an array of all the current stone positions
+// as a []*move.Move .
```

Comment:
Nit: you don't need to specify the type in the comments because it's in the method signature.